sfm/reconstruction.py

```diff
@@ -97,21 +97,22 @@ def poisson(self, depth=8, width=0, scale=1.25, l_fit=False, pcd=None):
 
         
 
-    def bpa(self, pcd=None, radius=None):
+    def bpa(self, pcd=None, radii=None):
         # assumes normals in poly
         if pcd is None:
             pcd = self.pcd
         
-        if radius is None:
+        if radii is None:
             D = pcd.compute_nearest_neighbor_distance()
             avg_D = np.mean(D)
             radius = 3 * avg_D
+            radii = [radius, radius * 2]
         
         radius = np.float64(radius)
```

Comment:
this line is no longer necessary

Reply:
im talking about "radius = np.float64(radius)"

Reply:
o word good now?

Reply:
looks good